### DISCUSSION TITLE
Send Message Hangs

### DISCUSSION THREAD

**Discussion by achrafmam2** [aldinokemal/go-whatsapp-web-multidevice#212] [General]
When I try to send a private message from the UI or through Curl it hangs (nothing happens). I tried it from 2 different computers and 2 different internet endpoints.

My setup is a MacBook Air M3. I am using my personal WA phone number.

Error I got:

```
Error: failed to get device list: failed to send usync query: info query timed out
```

**Comment by aldinokemal** [comment]
It seems to be a connection issue. You can try with a different connection.
